README.md

```diff
@@ -8,17 +8,28 @@ It was designed for tight integration with [Pyserini](http://pyserini.io/), but
 
 Currently, this repo contains implementations of the rerankers for [CovidQA](https://github.com/castorini/pygaggle/blob/master/data/) on CORD-19, as described in ["Rapidly Bootstrapping a Question Answering Dataset for COVID-19"](https://arxiv.org/abs/2004.11339).
 
+## Usage
+
+0. Install via PyPI `pip install pygaggle`. Requires [Python 3.6+](https://www.python.org/downloads/)
```

Comment:
I don't have context on what a simple usage might be. Hoping we can add something similar to the example in https://github.com/castorini/pyserini